### ISSUE TITLE
Return statement closing bracket position is not linted by indent rule

### ISSUE THREAD

**Issue by eelyafi** [eslint/eslint#7164]
**Tell us about your environment**

* **ESLint Version: master, 3.5.0 the latest at this moment**
* **Node Version: v6.2.1**
* **npm Version: 3.9.3**

* **parser: default**

Code to test:

```js
function() {
  return (
     1
    );
}
```

Issue:
position of ");" is off by 2 extra spaces, but linter didn't raise an error

Raw output:
```
1 failing

  1) indent invalid function foo() {
  return (
    1
    );
}:

      AssertionError: Should have 1 error but had 0: []
      + expected - actual

      -0
      +1
```

**Comment by eelyafi** [comment]
I think I have a fix for this one

**Comment by not-an-aardvark** [comment]
Your "code to test" sample is displaying like [this](https://i.gyazo.com/6080e2b643bc59bf59a9ac07aba936e8.png) for me, which makes it hard to figure out what indentation you're using. Would you mind putting that in a code block like you did with the 'raw output' section?

**Comment by eelyafi** [comment]
@not-an-aardvark sorry for the confusion, I updated the code snippet

**Comment by platinumazure** [comment]
Confirmed with this snippet (no lint error on line 6 but there should be):

```js
/* eslint indent:["error", 2] */

function a() {
  return (
     1
    );
}
```

@eelyafi Please let us know if you need any help working on a fix, or if you won't be able to work on a fix after all so someone else might be able to take over. Thanks!
